source/clear-linux/guides/maintenance/kernel-modules.rst

```diff
@@ -0,0 +1,192 @@
+.. _kernel-modules:
```

Comment:
Title needs to be more focused. Suggest: Loading an out-of-tree Kernel Module.

Reply:
As a result, I recommend a new intro with th explanation of kernel modules along side a description of what the article is going to show you how to do.

Reply:
> Title needs to be more focused. Suggest: Loading an out-of-tree Kernel Module.

Please change title to "Load an out-of-tree kernel module" for consistency with other help topics (no -ing words and sentence case).

Reply:
> Title needs to be more focused. Suggest: Loading an out-of-tree Kernel Module.

I'm not sure I agree for the top level title. The document is broader about kernel modules in CL. Loading out of tree modules is just one solution in the document.